includes/class-sensei-lesson.php

```diff
@@ -2245,10 +2245,12 @@ public function add_column_data( $column_name, $id ) {
 				} // End If Statement
 				break;
 			case 'lesson-prerequisite':
-				$lesson_prerequisite_id = get_post_meta( $id, '_lesson_prerequisite', true );
+				$lesson_prerequisite_id   = get_post_meta( $id, '_lesson_prerequisite', true );
+				$lesson_prerequisite_post = get_post( $lesson_prerequisite_id );
```

Comment:
This should be moved inside the `if` statement on line 2250 as it's not necessary to call this function if the lesson doesn't have a prerequisite.

Reply:
Done!